src/utils/aggregation/buildPipeline.ts

```diff
@@ -58,6 +58,19 @@ const buildPipeline = (
         break;
       }
       case PipelineStage.SORT: {
+        // Try to group sort stages together ( when they are following )
+        if (pipeline.length > 0) {
```

Comment:
@estelafs 
the logic was working, but I changed it so only consecutive stages will group ( in order to avoid some unwanted behaviors )